test/extended/builds/pipeline_origin_bld.go

```diff
@@ -272,6 +273,38 @@ var _ = g.Describe("[Feature:Builds][Feature:Jenkins][Slow] openshift pipeline b
 				o.Expect(err).NotTo(o.HaveOccurred())
 			})
 
+			g.By("should verify services successfully", func() {
+				// instantiate the bc
+				g.By("create the jenkins pipeline strategy build config that leverages openshift client plugin")
+				err := oc.Run("new-app").Args("-f", verifyServiceClientPluginPipelinePath).Execute()
+				o.Expect(err).NotTo(o.HaveOccurred())
+
+				err = exutil.WaitForDeploymentConfig(oc.KubeClient(), oc.AppsClient().AppsV1(), oc.Namespace(), "nodejs-example", 1, false, oc)
+				if err != nil {
+					exutil.DumpApplicationPodLogs("nodejs-example", oc)
+				}
+
+				brservices, err := exutil.StartBuildAndWait(oc, "jenkins-verifyservice-pipeline")
+				debugAnyJenkinsFailure(brservices, oc.Namespace()+"-jenkins-verifyservice-pipeline", oc, true)
```

Comment:
move this into the error block

admittedly you copied this from some other of the tests ... they should have moved it within the err block as well ... feel free to correct those if you like @waveywaves but I'm good either way

Reply:
I have moved all debugAnyJenkinsFailure calls to the respective error blocks as you suggested.